storage/aptosdb/src/pruner/mod.rs

```diff
@@ -77,7 +77,19 @@ impl Pruner {
     ) -> Self {
         let (command_sender, command_receiver) = channel();
 
-        let min_readable_version = Arc::new(Mutex::new(vec![0, 0, 0, 0, 0]));
+        let min_readable_version = Arc::new(Mutex::new(vec![
+            if storage_pruner_config.state_store_prune_window.is_none() {
```

Comment:
nit: This can be shortened for something like. 

```
storage_pruner_config.state_store_prune_window.map(|_| 0) 
```

Reply:
Done